tests/roxctl/bats-tests/local/roxctl-netpol-connectivity-map-development.bats

```diff
@@ -437,19 +437,7 @@ payments/gateway[Deployment] => payments/visa-processor[Deployment] : TCP 8080'
   assert_file_exist "$ofile"
   # normalizing tabs and whitespaces in output so it will be easier to compare with expected
   output=$(normalize_whitespaces "$output")
-  expected_output='backend/checkout[Deployment] => backend/notification[Deployment] : TCP 8080
-backend/checkout[Deployment] => backend/recommendation[Deployment] : TCP 8080
-backend/checkout[Deployment] => payments/gateway[Deployment] : TCP 8080
-backend/recommendation[Deployment] => backend/catalog[Deployment] : TCP 8080
-backend/reports[Deployment] => backend/catalog[Deployment] : TCP 8080
-backend/reports[Deployment] => backend/recommendation[Deployment] : TCP 8080
-frontend/webapp[Deployment] => backend/checkout[Deployment] : TCP 8080
-frontend/webapp[Deployment] => backend/recommendation[Deployment] : TCP 8080
-frontend/webapp[Deployment] => backend/reports[Deployment] : TCP 8080
-frontend/webapp[Deployment] => backend/shipping[Deployment] : TCP 8080
-payments/gateway[Deployment] => payments/mastercard-processor[Deployment] : TCP 8080
-payments/gateway[Deployment] => payments/visa-processor[Deployment] : TCP 8080
-{ingress-controller} => frontend/asset-cache[Deployment] : TCP 8080
+  partial_expected_output='{ingress-controller} => frontend/asset-cache[Deployment] : TCP 8080
```

Comment:
existing tests from main had to change?

Reply:
here made it patial output by removing some of the connlist output and focusing on the exposure output

Reply:
since the files are not large now (after removing new tests with formats other than txt) i can revert changes to old tests; to do that?

Reply:
yes, please